CELLDB.md

```diff
@@ -0,0 +1,56 @@
+Cell database management
```

Comment:
zou dit niet gewoon in de README erbij kunnen?

Reply:
Goede suggestie; ik wil het nu nog even apart laten omdat het niet direct past in de opzet van README